packages/datahub-data/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@colonial-collections/datahub-data",
```

Comment:
Or should we name it e.g. `database-connector` or just `database`? The name `datahub-data` could suggest that the linked data from the triplestores is also a part of this package.

Reply:
Decided in a meeting on `database`. Still very vague, but for now we don't know the scope of this package.